src/main/java/de/qabel/desktop/storage/HttpReadBackend.java

```diff
@@ -36,34 +36,34 @@ public StorageDownload download(String name, String ifModifiedVersion) throws Qb
 		}
 		prepareRequest(httpGet);
 
-		CloseableHttpResponse response;
 		try {
-			response = httpclient.execute(httpGet);
-		} catch (IOException e) {
-			throw new QblStorageException(e);
-		}
-		int status = response.getStatusLine().getStatusCode();
-		if ((status == 404) || (status == 403)) {
-			throw new QblStorageNotFound("File not found");
-		}
-		if (status == HttpStatus.SC_NOT_MODIFIED) {
-			throw new UnmodifiedException();
-		}
-		if (status != 200) {
-			throw new QblStorageException("Download error");
-		}
-		String modifiedVersion = response.getFirstHeader(HttpHeaders.ETAG).getValue();
+			CloseableHttpResponse response = httpclient.execute(httpGet);
+			try {
+				int status = response.getStatusLine().getStatusCode();
+				if ((status == 404) || (status == 403)) {
+					throw new QblStorageNotFound("File not found");
+				}
+				if (status == HttpStatus.SC_NOT_MODIFIED) {
```

Comment:
Why do only use the HttpStatus constants here and not consistently everywhere or never?